coalib/bearlib/abstractions/Linter.py

```diff
@@ -497,15 +517,15 @@ def process_output(self, output, filename, file):
 
         @classmethod
         @contextmanager
-        def _create_config(cls, filename, file, **kwargs):
+        def _create_config(cls, filename=None, file=None, **kwargs):
             """
             Provides a context-manager that creates the config file if the
             user provides one and cleans it up when done with linting.
 
             :param filename:
-                The filename of the file.
+                The filename of the file. None for project scope.
             :param file:
-                The file contents.
+                The file contents. None for project scope.
```

Comment:
could you enclose all `None`s in double backticks so they get rendered as code? (`None` is a code keyword, so I think we should render it appropriately 😄  )